### ISSUE TITLE
Uncaught TypeError: this.layer.translateToBufferMarkerParams is not a function

### ISSUE THREAD

**Issue by btd** [atom-minimap/minimap-cursorline#2]
**Atom Version**: 1.9.0-dev-57c17fb
**System**: Mac OS X 10.11.5
**Thrown From**: [minimap-cursorline](https://github.com/atom-minimap/minimap-cursorline) package, v0.1.0

### Stack Trace

Uncaught TypeError: this.layer.translateToBufferMarkerParams is not a function

```
At /Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/display-marker.js:98

TypeError: this.layer.translateToBufferMarkerParams is not a function
    at DisplayMarker.module.exports.DisplayMarker.matchesProperties (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/display-marker.js:98:31)
    at /Users/den/.atom/packages/minimap-cursorline/lib/minimap-cursorline-binding.js:16:18
    at TextEditor.module.exports.TextEditor.observeCursors (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:467:9)
    at new MinimapCursorLineBinding (/Users/den/.atom/packages/minimap-cursorline/lib/minimap-cursorline-binding.js:14:40)
    at /Users/den/.atom/packages/minimap-cursorline/lib/minimap-cursorline.js:40:23
    at /Users/den/.atom/packages/minimap/lib/main.js:352:51
    at Map.forEach (native)
    at Main.observeMinimaps (/Users/den/.atom/packages/minimap/lib/main.js:352:28)
    at Object.activatePlugin (/Users/den/.atom/packages/minimap-cursorline/lib/minimap-cursorline.js:34:46)
    at Main.activatePlugin (/Users/den/.atom/packages/minimap/lib/mixins/plugin-management.js:182:12)
    at Main.updatesPluginActivationState (/Users/den/.atom/packages/minimap/lib/mixins/plugin-management.js:166:14)
    at /Users/den/.atom/packages/minimap/lib/mixins/plugin-management.js:234:12
    at Config.module.exports.Config.observeKeyPath (/Applications/Atom.app/Contents/Resources/app.asar/src/config.js:612:7)
    at Config.module.exports.Config.observe (/Applications/Atom.app/Contents/Resources/app.asar/src/config.js:141:21)
    at Main.registerPluginControls (/Users/den/.atom/packages/minimap/lib/mixins/plugin-management.js:233:53)
    at Main.registerPlugin (/Users/den/.atom/packages/minimap/lib/mixins/plugin-management.js:78:12)
    at Object.consumeMinimapServiceV1 (/Users/den/.atom/packages/minimap-cursorline/lib/minimap-cursorline.js:19:18)
    at Provider.module.exports.Provider.provide (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/service-hub/lib/provider.js:30:52)
    at /Applications/Atom.app/Contents/Resources/app.asar/node_modules/service-hub/lib/service-hub.js:55:26
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
    at process._tickCallback (internal/process/next_tick.js:98:9)
```

### Commands

```
```

### Config

```json
{
  "core": {}
}
```

### Installed Packages

```coffee
# User
atom-jinja2, v0.5.0 (inactive)
css-snippets, v0.9.0 (inactive)
docblockr, v0.7.3 (inactive)
formatter, v2.12.3 (inactive)
highlight-selected, v0.11.2 (inactive)
language-babel, v2.24.4 (inactive)
language-bison, v1.0.0 (inactive)
language-docker, v1.1.6 (inactive)
language-groovy, v0.6.0 (inactive)
language-scala, v1.1.6 (inactive)
lcov-info, v0.11.0 (inactive)
linter, v1.11.4 (inactive)
linter-eslint, v7.2.2 (inactive)
minimap, v4.24.3 (inactive)
minimap-bookmarks, v0.3.1 (inactive)
minimap-cursorline, v0.1.0 (inactive)
minimap-find-and-replace, v4.5.1 (inactive)
minimap-git-diff, v4.3.1 (inactive)
minimap-highlight-selected, v4.4.0 (inactive)
minimap-linter, v1.1.1 (inactive)
minimap-pigments, v0.2.1 (inactive)
minimap-selection, v4.4.0 (inactive)
pigments, v0.29.0 (inactive)
split-diff, v1.0.0 (inactive)
todo-show, v1.6.0 (inactive)
tool-bar, v0.4.0 (inactive)
atom-dark-syntax, v0.27.0 (inactive)
atom-dark-ui, v0.51.0 (inactive)
atom-light-syntax, v0.28.0 (inactive)
atom-light-ui, v0.43.0 (inactive)
base16-tomorrow-dark-theme, v1.1.0 (inactive)
base16-tomorrow-light-theme, v1.1.1 (inactive)
one-dark-ui, v1.3.1 (active)
one-light-ui, v1.3.1 (inactive)
one-dark-syntax, v1.2.0 (active)
one-light-syntax, v1.2.0 (inactive)
solarized-dark-syntax, v1.0.2 (inactive)
solarized-light-syntax, v1.0.2 (inactive)
about, v1.5.2 (inactive)
archive-view, v0.61.1 (inactive)
autocomplete-atom-api, v0.10.0 (inactive)
autocomplete-css, v0.11.1 (inactive)
autocomplete-html, v0.7.2 (inactive)
autocomplete-plus, v2.31.0 (inactive)
autocomplete-snippets, v1.11.0 (inactive)
autoflow, v0.27.0 (inactive)
autosave, v0.23.1 (inactive)
background-tips, v0.26.0 (inactive)
bookmarks, v0.41.0 (inactive)
bracket-matcher, v0.82.1 (inactive)
command-palette, v0.38.0 (inactive)
deprecation-cop, v0.54.1 (inactive)
dev-live-reload, v0.47.0 (inactive)
encoding-selector, v0.22.0 (inactive)
exception-reporting, v0.38.1 (inactive)
fuzzy-finder, v1.2.0 (inactive)
git-diff, v1.0.1 (inactive)
find-and-replace, v0.198.0 (inactive)
go-to-line, v0.31.0 (inactive)
grammar-selector, v0.48.1 (inactive)
image-view, v0.57.0 (inactive)
incompatible-packages, v0.26.1 (inactive)
keybinding-resolver, v0.35.0 (inactive)
line-ending-selector, v0.5.0 (inactive)
link, v0.31.1 (inactive)
markdown-preview, v0.158.0 (inactive)
metrics, v0.53.1 (inactive)
notifications, v0.64.1 (inactive)
open-on-github, v1.1.0 (inactive)
package-generator, v1.0.0 (inactive)
settings-view, v0.238.0 (inactive)
snippets, v1.0.2 (inactive)
spell-check, v0.67.1 (inactive)
status-bar, v1.2.6 (inactive)
styleguide, v0.45.2 (inactive)
symbols-view, v0.113.0 (inactive)
tabs, v0.97.0 (inactive)
timecop, v0.33.1 (inactive)
tree-view, v0.208.0 (inactive)
update-package-dependencies, v0.10.0 (inactive)
welcome, v0.34.0 (inactive)
whitespace, v0.32.2 (inactive)
wrap-guide, v0.38.1 (inactive)
language-c, v0.52.1 (inactive)
language-clojure, v0.21.0 (inactive)
language-coffee-script, v0.47.0 (inactive)
language-csharp, v0.12.1 (inactive)
language-css, v0.36.1 (inactive)
language-gfm, v0.86.0 (inactive)
language-git, v0.13.0 (inactive)
language-go, v0.42.0 (inactive)
language-html, v0.44.1 (inactive)
language-hyperlink, v0.16.0 (inactive)
language-java, v0.21.0 (inactive)
language-javascript, v0.110.0 (inactive)
language-json, v0.18.0 (inactive)
language-less, v0.29.3 (inactive)
language-make, v0.22.0 (inactive)
language-mustache, v0.13.0 (inactive)
language-objective-c, v0.15.1 (inactive)
language-perl, v0.35.0 (inactive)
language-php, v0.37.0 (inactive)
language-property-list, v0.8.0 (inactive)
language-python, v0.45.0 (inactive)
language-ruby, v0.68.5 (inactive)
language-ruby-on-rails, v0.25.0 (inactive)
language-sass, v0.52.0 (inactive)
language-shellscript, v0.22.3 (inactive)
language-source, v0.9.0 (inactive)
language-sql, v0.21.1 (inactive)
language-text, v0.7.1 (inactive)
language-todo, v0.28.0 (inactive)
language-toml, v0.18.0 (inactive)
language-xml, v0.34.6 (inactive)
language-yaml, v0.26.0 (inactive)

# Dev
No dev packages
```

**Comment by fundon** [comment]
Currently The stable version is 1.7.x, so when the 1.9.x is coming in. We will check this. Thanks.

**Comment by rgrove** [comment]
1.9.0 just hit the beta channel, so a lot more people are going to encounter this bug.

**Comment by fundon** [comment]
@rgrove @btd  Ok, thanks to your reports.
We are fixing it.

**Comment by fundon** [comment]
Landed in v0.2.0.
